src/main/java/uk/gov/hmcts/reform/sandl.snlevents/converters/OffsetDateTimeAttributeConverter.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sandl.snlevents.converters;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+@Converter(autoApply = true)
+public class OffsetDateTimeAttributeConverter implements AttributeConverter<OffsetDateTime, Timestamp> {
```

Comment:
What is it for?

Reply:
new version of hibernate no longer works with OffsetDateTime unless converted used, not sure why they changed it - similar to what we needed to do with duration.